packages/server-core/src/user/strategies/github.ts

```diff
@@ -128,28 +128,32 @@ export class GithubStrategy extends CustomOAuthStrategy {
     }
   }
 
-  async getRedirect(data: any, params: CustomOAuthParams): Promise<string> {
-    const redirectHost = config.authentication.callback.github
```

Comment:
@barankyle since we are no longer using `config.authentication.callback.github` and same for other strategies, how is it going to impact magic-link functionality?